src/helpers/tests/resultSpec.spec.ts

```diff
@@ -193,3 +193,45 @@ describe('getStackingResultSpec', () => {
         expect(getStackingResultSpec(buckets, [measureSortItem])).toEqual(expectedResultSpec);
     });
 });
+
+describe('getGeneralResultSpec', () => {
+    const measure = {
+        alias: 'Measure m1',
+        definition: {
+            measure: {
+                item: {
+                    identifier: 'ident_m1'
+                }
+            }
+        },
+        localIdentifier: 'm1'
+    };
+    const attribute = {
```

Comment:
please use `ATTRIBUTE_1` imported above instead of creating new variable

Reply:
AFM.IAttribute is different from VisualizationObject.IAttribute. This is why this is not ATTRIBUTE_1